scripts/babel/wrap-warning-with-env-check.js

```diff
@@ -1,67 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-'use strict';
-
-module.exports = function(babel, options) {
-  const t = babel.types;
-
-  const DEV_EXPRESSION = t.identifier('__DEV__');
-
-  const SEEN_SYMBOL = Symbol('expression.seen');
-
-  return {
-    visitor: {
-      CallExpression: {
-        exit: function(path) {
-          const node = path.node;
-
-          // Ignore if it's already been processed
-          if (node[SEEN_SYMBOL]) {
-            return;
-          }
-
-          if (
-            path.get('callee').isIdentifier({name: 'warning'}) ||
-            path.get('callee').isIdentifier({name: 'warningWithoutStack'})
-          ) {
-            // Turns this code:
-            //
-            // warning(condition, argument, argument);
-            //
-            // into this:
-            //
-            // if (__DEV__) {
-            //   if (!condition) {
```

Comment:
This part of the babel plugin is no longer in the ESLint rule causing the dev packages to have the following diff:

```diff
- !(typeof trustedTypes === 'undefined') ? warning$1(false, "Using 'dangerouslySetInnerHTML' in an svg element with " + 'Trusted Types enabled in an Internet Explorer will cause ' + 'the trusted value to be converted to string. Assigning string ' + "to 'innerHTML' will throw an error if Trusted Types are enforced. " + "You can try to wrap your svg element inside a div and use 'dangerouslySetInnerHTML' " + 'on the enclosing div instead.') : void 0;
+ warning$1(typeof trustedTypes === 'undefined', "Using 'dangerouslySetInnerHTML' in an svg element with " + 'Trusted Types enabled in an Internet Explorer will cause ' + 'the trusted value to be converted to string. Assigning string ' + "to 'innerHTML' will throw an error if Trusted Types are enforced. " + "You can try to wrap your svg element inside a div and use 'dangerouslySetInnerHTML' " + 'on the enclosing div instead.');
```